src/netex_idf/mod.rs

```diff
@@ -20,6 +20,7 @@ mod lines;
 mod modes;
 mod offers;
 mod read;
+mod share;
```

Comment:
How about calling it `accessibility` instead of `share`?

Reply:
i you want. I was planning to add more things